src/server/wtr-config.js

```diff
@@ -14,6 +14,7 @@ const BROWSER_MAP = {
 	safari: 'webkit',
 	webkit: 'webkit'
 };
+const DEFAULT_VDIFF_SLOW = 500;
```

Comment:
I'm guessing we'll want to tweak this later once we have vdiff running in core and have a better idea of what "slow" means for visual-diff. We may also want different values locally compared to CI.